gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapperTest.java

```diff
@@ -289,6 +307,72 @@ public void mapConditionWithSnomedCodeInCoding() {
         assertEquals(codeableConcept, conditions.get(0).getCode());
     }
 
+    @Test
+    void When_Condition_With_NopatConfidentialityCode_Expect_MetaFromConfidentialityServiceWithSecurity() {
+        final Meta metaWithSecurity = MetaFactory.getMetaFor(META_WITH_SECURITY, META_PROFILE);
+        final RCMRMT030101UKEhrExtract ehrExtract =
+            unmarshallEhrExtract("linkset_valid_nopat_confidentiality_code.xml");
+
+        when(dateTimeMapper.mapDateTime(
+            any(String.class)
+        )).thenReturn(EHR_EXTRACT_AVAILABILITY_DATETIME);
+
+        when(confidentialityService.createMetaAndAddSecurityIfConfidentialityCodesPresent(
+            eq(META_PROFILE),
+            confidentialityCodeCaptor.capture(),
+            confidentialityCodeCaptor.capture()
+        )).thenReturn(MetaFactory.getMetaFor(META_WITH_SECURITY, META_PROFILE));
+
+        final List<Condition> conditions = conditionMapper
+            .mapResources(ehrExtract, patient, Collections.emptyList(), PRACTISE_CODE);
+
+        conditionMapper.addReferences(buildBundleWithNamedStatementObservation(), conditions, ehrExtract);
```

Comment:
Is calling this method important, I think the production code we care about is within `mapResources` which is called on the line above. I suspect calling this `addReferences` has no effect on the outcome of the test, and is just a distraction.